core/filenumber_checker.py

```diff
@@ -0,0 +1,40 @@
+from typing import List
+from pathlib import Path
+
+import pandas as pd
+from moviepy.video.io.VideoFileClip import VideoFileClip
```

Comment:
moviepy is a new dependency, that would be needed to run the pipeline. 
it should be installed with DLC? anyways, but maybe we can use imageio instead to stay consistent?